localstack-core/localstack/services/lambda_/event_source_mapping/esm_worker.py

```diff
@@ -148,9 +172,10 @@ def poller_loop(self, *args, **kwargs):
                     e,
                     exc_info=LOG.isEnabledFor(logging.DEBUG),
                 )
-                # TODO: implement some backoff here and stop poller upon continuous errors
```

Comment:
strictly speaking, we could parity test when pollers are shut down at AWS after continous errors. However, I think the current solution with the up to 60s max error backoff is good enough (or maybe even preferrable in a dev environment to make the error visible).